pkg/config/config.go

```diff
@@ -0,0 +1,60 @@
+package config
+
+import (
+	"encoding/json"
+	"github.com/GoSome/fileUpdater/pkg/types"
+	"github.com/fsnotify/fsnotify"
+	"gopkg.in/yaml.v3"
+	"log"
+	"os"
+	"strings"
+	"sync"
+)
+
+var Path string
+var Configs types.ServerConfigs
+var lock sync.Mutex
+
+func Load() {
+	if _, err := os.Stat(Path); os.IsNotExist(err) {
+		log.Fatalf("config file \"%s\" not exist", Path)
+	}
+
+	configFile, err := os.Open(Path)
+	if err != nil {
+		log.Fatalf("open config file \"%s\" failed", Path)
```

Comment:
load() should think about run stats, when first start.it can use panic,but when hot reload should not use panic,  i think It's better to have an config validator